### PR TITLE
refactor: Replace `_.flatMap` with `Array.prototype.flatMap`

### PR DESCRIPTION
**Pull request by pjmattingly**
Addresses: #7747

### CODE REVIEW

**Comment by pjmattingly** [comment]
When addressing #7747 for _.flatmap, I noted that there were additional lodash calls that seemed similar. Both _.flattenDeep() and _.flattenDepth() can be replaced with Array.prototype.flat(); And so I took the liberty of making those small changes as well.

File: lib/plugins/aws/invoke-local/index.js [hunk 1]
```diff
@@ -543,6 +543,7 @@ class AwsInvokeLocal {
       configuredEnvVars,
       envVarsFromOptions
     );
+    // #7747, envVars is not Array, and so we cannot use Array.prototype.flatMap() here
     const envVarsDockerArgs = _.flatMap(envVars, (value, key) => ['--env', `${key}=${value}`]);
```
Comment (pgrzesik) [inline lib/plugins/aws/invoke-local/index.js:547]: Maybe we could still replace it with different calls, e.g. `Object.entries` on `envVars` to change it into an array?

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thanks @pjmattingly 🙇 I have one comment, please let me know what do you think

File: lib/plugins/aws/invoke-local/index.js (hunk 1 above)
Comment (pjmattingly) [reply lib/plugins/aws/invoke-local/index.js:547]: That worked, though note this small change:

```
const envVarsDockerArgs = _.flatMap(envVars, (value, key) => ['--env', `${key}=${value}`]);
```

to

```
const envVarsDockerArgs = Object.entries(envVars).flatMap( (currentValue) => ['--env', `${currentValue[0]}=${currentValue[1]}`])
```

`array.flatMap()` passes in the values from `entries()` as a single array (`currentValue`). So, `key` and `value` are then indices in the `currentValue` array.

see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#syntax

Comment (pgrzesik) [reply lib/plugins/aws/invoke-local/index.js]: Thanks for clarification, I think that works well and allows to replace `_.flatMap` 👍

**Review by pgrzesik — APPROVED** [review]
Well done @pjmattingly 👏 Thank you 🙇

**Pull request merged**